packages/screenreaderonly/src/css/index.css

```diff
@@ -0,0 +1,11 @@
+.psds-screenreader-only {
```

Comment:
this also exists in core. Might be a reuse opp, one way or the other. I think I'd prefer the core usage. Probably not this PR, though. What do you think?